### ISSUE TITLE
Can oscillation prototype code be removed from the UI?

### ISSUE THREAD

**Issue by jbphet** [phetsims/tambo#85]
@samreid - some prototyping of sound ideas was done a while back for Wave Interference, and I think it may be possible to remove it at this point.  Can it go?  It's on the first screen of the tambo demo and looks like this:

![image](https://user-images.githubusercontent.com/3082831/69269713-dbda4d80-0b8e-11ea-8d21-c2cc2d1d24d1.png)

**Comment by samreid** [comment]
We should first decide what to do about ForceSoundGenerator and ContinuousPropertySoundGenerator.  In https://github.com/phetsims/wave-interference/issues/436#issuecomment-522740376 it was recommended to create a common code component ContinuousPropertySoundGenerator.  I created this based on ForceSoundGenerator from gravity-force-lab.  I would need help fully generalizing ContinuousPropertySoundGenerator so that it could be used in gravity-force-lab and so that ForceSoundGenerator could be deleted.  If we decide to keep ContinuousPropertySoundGenerator or something like it, it seems that would be nice to have a test harness in the tambo demo.

**Comment by jbphet** [comment]
> We should first decide what to do about ForceSoundGenerator and ContinuousPropertySoundGenerator.

I did the generalization and have replaced `ForceSoundGenerator` with `ContinuousPropertySoundGenerator` in both GFL and GFLB.  @samreid - please review my changes and make sure they don't mess anything up for your current usage.  I don't think they will, since I mostly added new options, but I did change the mapping algorithm to remove the MIDI note references (since I think such reference won't make sense to most developers), so there's a chance the pitch range could have changed.

> [I]t seems that would be nice to have a test harness in the tambo demo.

Agreed, but not four of them at the upper left of the first screen.  How about having one, and putting it on the UI Components screen, perhaps in the currently available space at the lower left.  Eventually, we should do something like what was done in the sun repo, where there is a combo box that can be used to select the various components, but that's a low priority at this point.

I had two other thoughts that I'd like your (@samreid's) opinion on:
+ Should we make the value-to-pitch mapping algorithm an option that can be provided if needed, defaulting to the original behavior?
+ Shall we make `resetInProgressProprty` an option with a more general name?  I think we will need this behavior in many cases, but not all.

**Comment by samreid** [comment]
The changes look great, thanks!  One comment could be improved, it currently reads:

```js
// calculate the playback rate based on the amount of force, see the design document for detailed explanation
```

ContinuousPropertySoundGenerator probably shouldn't refer to force or a design document.

> How about having one, and putting it on the UI Components screen, perhaps in the currently available space at the lower left. Eventually, we should do something like what was done in the sun repo, where there is a combo box that can be used to select the various components, but that's a low priority at this point.

That sounds great to me!

> Should we make the value-to-pitch mapping algorithm an option that can be provided if needed, defaulting to the original behavior?

That sounds like something we can defer until if/when we need it.

> Shall we make resetInProgressProprty an option with a more general name? I think we will need this behavior in many cases, but not all.

What pattern do we typically use for muting a sound during resets?  Can we use that strategy here?

**Comment by jbphet** [comment]
I've moved the set of panels to the "Test" screen since there is good infrastructure there for it now.  I also fixed the erroneous docs.  As for the reset handling, I've left it as is for now, since it is working fine in Gravity Force Lab and the name represents what it is at this point.

The demo isn't perfect because it doesn't handle resets, but none of the other ones do either and the demo screen superclass isn't set up very well for it.

I think I've adequately addressed everything that was raised above.  Closing.
